### PR TITLE
Add Plain

### PR DESCRIPTION
**Pull request by fmerian**
https://plain.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,6 +233,7 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 * [Interval](https://interval.com/) - SDK to build internal tools and scripts for your product.
 * [ngrok](https://ngrok.com/) - Generate public URLs for internal servers (behind NAT/firewall).
 * [Nylas](https://www.nylas.com/) - APIs for productivity workflows (email, calendar, contacts...) - like plaid for productivity.
+* [Plain](https://plain.com) - API-first customer service platform.
```
Comment (agamm) [inline README.md:236]:
```suggestion
* [Plain](https://plain.com) - API-first customer service platform (support, feedback, rating widgets...).
```

**Review by agamm — CHANGES_REQUESTED** [review]
What about something more concrete?

File: README.md (hunk 1 above)
Comment (fmerian) [reply README.md]: LGTM :)

**Review by agamm — APPROVED** [review]
LGTM :)

**Comment by fmerian** [comment]
ping @simonrohrbach @mattvagni for the record

**Pull request merged**